pages/k8s/release-notes.md

```diff
@@ -45,16 +45,69 @@ Existing deployments that are upgraded to **CDK 1.14** will
-xxxxxx xxx xxx xx xxxx xxxx  [documentation][doc-link].
 
 ## Fixes
 
-- Fixed xxxx xxx xx xx xxx xxxxxxx xxxx ([Issue](https://bugs.launchpad.net/charmed-kubernetes/))
-- Fixed xxxx xxx xxxxx xx xx xxxxx xxxx ([Issue](https://bugs.launchpad.net/charmed-kubernetes/))
-- Fixed xx x xxx x xx xxx xxx xxxx xx xx ([Issue](https://bugs.launchpad.net/charmed-kubernetes/))
+ - Added an action to upgrade Docker ([Issue](https://github.com/juju-solutions/layer-docker/pull/135))
+ - Added better multi-client support to EasyRSA ([Issue](https://github.com/juju-solutions/layer-easyrsa/pull/15))
+ - Added block storage config options ([Issue](https://github.com/juju-solutions/charm-openstack-integrator/pull/15))
+ - Added block storage options for OpenStack ([Issue](https://github.com/juju-solutions/kubernetes/pull/218))
+ - Added dashboard-auth config option to master ([Issue](https://github.com/juju-solutions/kubernetes/pull/222))
+ - Added dashboard login using kubeconfig ([Issue](https://github.com/juju-solutions/cdk-addons/pull/72))
+ - Added docker registry handling to master ([Issue](https://github.com/juju-solutions/kubernetes/pull/210))
+ - Added more TLS options to Docker registry ([Issue](https://github.com/CanonicalLtd/docker-registry-charm/pull/20))
+ - Added new folder/respool_path config for vSphere ([Issue](https://github.com/juju-solutions/charm-vsphere-integrator/pull/2))
+ - Added proxy support to Docker registry ([Issue](https://github.com/CanonicalLtd/docker-registry-charm/pull/21))
+ - Added read-only mode for Docker registry ([Issue](https://github.com/CanonicalLtd/docker-registry-charm/pull/22))
+ - Added token auth option for dashboard ([Issue](https://github.com/juju-solutions/cdk-addons/pull/77))
+ - Fixed `allow-privileged` not enabled when Ceph relation joins ([Issue](https://github.com/juju-solutions/kubernetes/pull/197))
+ - Fixed apt install source for VaultLocker ([Issue](https://github.com/juju-solutions/layer-vaultlocker/pull/3))
+ - Fixed CDK broken when using GPUs due to nvidia-docker2 dependency failure ([Issue](https://github.com/juju-solutions/bundle-canonical-kubernetes/issues/720))
+ - Fixed Ceph relation join not creating necessary pools ([Issue](https://github.com/juju-solutions/bundle-canonical-kubernetes/issues/631))
+ - Fixed Ceph volume provisioning fails with "No such file or directory" ([Issue](https://github.com/juju-solutions/bundle-canonical-kubernetes/issues/710))
+ - Fixed client certs not having the requested common name ([Issue](https://github.com/juju-solutions/layer-easyrsa/pull/19))
+ - Fixed create_client_cert never running ([Issue](https://github.com/juju-solutions/layer-easyrsa/pull/17))
+ - Fixed csi-rbdplugin-attacher crash loop when allow-privileged=auto or false ([Issue](https://github.com/juju-solutions/bundle-canonical-kubernetes/issues/674))
+ - Fixed dashboard CVE-2018-18264 ([Issue](https://github.com/juju-solutions/cdk-addons/pull/68))
```

Comment:
Already covered in a [prior release note](https://github.com/juju-solutions/kubernetes-docs/blob/27adcc19039f2a8f1ebe3bf7c427ea8dfa84fe18/pages/k8s/release-notes-historic.md#cve-2018-18264). Also this is a duplicate of "Added dashboard login using kubeconfig" above.